browser/brave_browser_process_impl.cc

```diff
@@ -108,6 +111,9 @@ BraveBrowserProcessImpl::BraveBrowserProcessImpl(StartupData* startup_data)
   g_browser_process = this;
   g_brave_browser_process = this;
 
+  // early initialize brave farbling
```

Comment:
why can't this just lazy load?

Reply:
fixed in https://github.com/brave/brave-core/pull/12234/commits/e937302cb6925c96fb2ea026788eb4409d2e325c